db/hints/manager.cc

```diff
@@ -759,7 +759,8 @@ bool manager::end_point_hints_manager::sender::send_one_file(const sstring& fnam
 
     try {
         commitlog::read_log_file(fname, manager::FILENAME_PREFIX, service::get_local_streaming_priority(), [this, secs_since_file_mod, &fname, ctx_ptr] (commitlog::buffer_and_replay_position buf_rp) mutable {
-            auto&& [buf, rp] = buf_rp;
+            auto& buf = buf_rp.buffer;
+            auto& rp = buf_rp.position;
```

Comment:
Shouldn't it be auto&& ?

Reply:
The right hand side is an lvalue, and in auto&& x = lvalue 'auto&&' resolves to 'auto&'.

Reply:
Right.